crates/rust-client/src/rpc/domain/account.rs

```diff
@@ -273,30 +274,32 @@ impl AccountProof {
         Ok(Self { account_witness, state_headers })
     }
 
+    /// Returns the account ID related to the account proof.
     pub fn account_id(&self) -> AccountId {
         self.account_witness.id()
     }
 
+    /// Returns the account header, if present.
     pub fn account_header(&self) -> Option<&AccountHeader> {
         self.state_headers.as_ref().map(|headers| &headers.account_header)
     }
 
+    /// Returns the storage header, if present.
     pub fn storage_header(&self) -> Option<&AccountStorageHeader> {
         self.state_headers.as_ref().map(|headers| &headers.storage_header)
     }
 
+    /// Returns the account code, if present.
     pub fn account_code(&self) -> Option<&AccountCode> {
         self.state_headers.as_ref().map(|headers| &headers.code)
     }
 
-    pub fn state_headers(&self) -> Option<&StateHeaders> {
-        self.state_headers.as_ref()
-    }
-
+    /// Returns the code commitment, if account code is present in the state headers.
     pub fn code_commitment(&self) -> Option<Digest> {
         self.account_code().map(AccountCode::commitment)
     }
 
+    /// Returns the current account commitment.
```

Comment:
nit: The phrasing sounds a bit odd in this one, what about something like
```suggestion
    /// Returns the current commitment of the account.
```

Reply:
Agree that reads better! Changed